### PR TITLE
docs: add stormkit guide

### PR DESCRIPTION
**Pull request by danielroe**
### 🔗 Linked issue

https://github.com/unjs/nitro/issues/71

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This adds some details to the stormkit zero-config deployment guide.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/deploy/providers/stormkit.md [hunk 1]
```diff
@@ -6,6 +6,11 @@
 Integration with this provider is possible with zero configuration. ([Learn More](/deploy/#zero-config-providers))
 :::
 
-::: warning Work in Progress
-Documentation for this section is not yet complete.
-:::
+Nitro supports [Stormkit](https://www.stormkit.io/) out of the box.
```
Comment (pi0) [inline docs/deploy/providers/stormkit.md:9]: There is already an alert above isn't this duplicate? I'm fine to reword alert if you like BTW :)

Comment (danielroe) [reply docs/deploy/providers/stormkit.md:9]: The alert looks nice. The problem with zero config is there is so little to document 😆

File: docs/deploy/providers/stormkit.md [hunk 2]
```diff
@@ -6,6 +6,11 @@
...
+
+1. Follow the steps to [create a new app](https://app.stormkit.io/apps/new) on Stormkit.
```
Comment (pi0) [inline docs/deploy/providers/stormkit.md:11]: Can you please put images somewhere outside of markdowns in docs/images maybe?

**Comment by pi0** [comment]
Thanks!

**Pull request merged**